coreos-base/afterburn/files/coreos-metadata.service

```
@@ -8,7 +8,9 @@ Restart=on-failure
 RestartSec=10
 Environment=COREOS_METADATA_OPT_PROVIDER=--cmdline
 ExecStart=/usr/bin/coreos-metadata ${COREOS_METADATA_OPT_PROVIDER} --attributes=/run/metadata/flatcar
+ExecStartPost=/usr/bin/sed --in-place "s/AFTERBURN/COREOS/ ; s/AWS/EC2/ ; s/GCP/GCE/" /run/metadata/flatcar
```

Should those be with the `g` flag in case there are more instances to replace?
```suggestion
ExecStartPost=/usr/bin/sed --in-place "s/AFTERBURN/COREOS/g ; s/AWS/EC2/g ; s/GCP/GCE/g" /run/metadata/flatcar
```

---

Good question - I think we can safely add the `g` flag. From what I seen in the codebase, it's always `AFTERBURN_$PROVIDER_$ATTRIBUTE`, so we _should_ not have redundancy on the same line.
However, it's better to preserve consistency if we want to avoid confusing situation while we maintain backward compatibility.